server/server.js

```diff
@@ -0,0 +1,6 @@
+const app = require('./index.js');
+const port = 3000;
+
```

Comment:
Why separate index and server?

Reply:
When I was debugging jest SO recommended I separate the app.listen into its own file so multiple listens don't happen.